components/org.wso2.carbon.identity.api.user.common/src/main/java/org/wso2/carbon/identity/api/user/common/Util.java

```diff
@@ -141,4 +142,27 @@ private static boolean validateUserIdInUserstore(RealmService realmService, Stri
         }
         return ((UniqueIDUserStoreManager) userStoreManager).isExistingUserWithID(userId);
     }
+
+    /**
+     * Validate whether the given filter is not empty and tenantDomain is valid.
+     *
+     * @param filter    filter to be applied for session termination
+     * @param tenantDomain  tenant domain of the requester
+     */
```

Comment:
Shall we fix the formatting errors here?

- Param descriptions are not aligned.
- Start sentences with uppercase letter and end with a `.`

Reply:
corrected.